wasmcloud_host/lib/wasmcloud_host_web/live/components/start_actor_component.ex

```diff
@@ -115,23 +137,35 @@ defmodule StartActorComponent do
 
   def render(assigns) do
     modal_id =
-      if assigns.id == :start_actor_file_modal do
-        "start_actor_file"
-      else
-        "start_actor_ociref"
+      case assigns.id do
+        :start_actor_file_modal -> "start_actor_file"
+        :start_actor_ociref_modal -> "start_actor_ociref"
+        :start_actor_file_hotreload_modal -> "start_actor_file_hotreload"
       end
 
     ~L"""
     <form class="form-horizontal" phx-submit="<%= modal_id %>" phx-change="validate" phx-target="<%= @myself %>">
       <input name="_csrf_token" type="hidden" value="<%= Phoenix.Controller.get_csrf_token() %>">
+      <%= if assigns.id == :start_actor_file_hotreload_modal do %>
+      <div class="form-group row" phx-drop-target="<%= @uploads.actor.ref %>">
+        <label class="col-md-3 col-form-label" for="file-input">Path</label>
+        <div class="col-md-9">
+          <input class="form-control" id="text-input" type="text" name="path"
```

Comment:
- I'm curious - why a text field and not `live_file_input`, as used in line 164 for upload actor, which provides drag & drop capability and is made for file paths?

Reply:
The only reason here is that live_file_input ends up creating a copy of the actor in the temp directory, which of course isn't repeated for actor changes, which we need to do a seamless watch for file events. If I missed a way to do that in Phoenix that ends up getting around the browser restriction that prevents this, I'd be happy to change that. Originally I envisioned this feature being a checkbox on the "From File" modal.